jet_tracking/jet_tracking_cal/jt_cal.py

```diff
@@ -0,0 +1,441 @@
+    )
+
+    peak_line = Span(location=peak, dimension='height', \
+        line_color='green', line_width=2)
+    lower_line = Span(location=peak-delta_bin, dimension='height', \
+        line_color='black')
+    upper_line = Span(location=peak+delta_bin, dimension='height', \
+        line_color='black')
+    ave_azav_curve = fig.scatter(x_vals, ave_azav)
+    fig.renderers.extend([peak_line, lower_line, upper_line])
+
+    azav_legend = Legend(items=[
+        LegendItem(label='Azimuthal Average', renderers=[ave_azav_curve])
+    ])
+    fig.add_layout(azav_legend)
+
+    return fig
+
+def intensity_hist(intensity_hist, edges):
+    fig = figure(
+            title='Intensity Histogram'
+    )
+    fig.quad(top=intensity_hist, bottom=0, left=edges[:-1], right=edges[1:])
+
+    return fig
+
+def intensity_vs_peak_fig(intensity, peak_vals, x, y, slope, intercept, sigma):
+    """Simple plot of intensity vs peak value"""
+    fig = figure(
+        title='Peak value vs Intensity. Slope = {0}, Intercept = {1}'.\
+            format(round(slope, 2), round(intercept, 2)),
+        x_axis_label='Intensity Monitor Value',
+        y_axis_label='Peak Values'
+    )
+    fig.x_range.range_padding = fig.y_range.range_padding = 0
+    h, y_edge, x_edge = np.histogram2d(peak_vals, intensity, bins=100)
+    fig.image(image=[h], x=x_edge[0], y=y_edge[0], dh=y_edge[-1]-y_edge[0], \
+        dw=x_edge[-1]-x_edge[0], palette="Spectral11")
+    color_mapper = LinearColorMapper(palette="Spectral11", low=h.min(), high=h.max())
+    color_bar = ColorBar(color_mapper=color_mapper, location=(0,0))
+    fig.xaxis.bounds = [i0_low, i0_high]
+    fig.add_layout(color_bar, 'right')
+    fig.line(x, y, color='red')
+    fig.line(x, y - 1 * sigma, color='orange')
+    fig.line(x, y + 1 * sigma, color='orange')
+    return fig
+
+if __name__ == '__main__':
+    parser = ArgumentParser()
+    parser.add_argument('--exp', type=str, \
+        default=(os.environ.get('EXPERIMENT', None)))
+    parser.add_argument('--run', type=str, \
+        default=(os.environ.get('RUN_NUM', None)))
+    parser.add_argument('--cfg', type=str, \
+        default=(''.join([JT_LOC, 'mpi_scripts/mpi_configs/default_config.yml'])))
+    parser.add_argument('--delta_bin', type=int, \
+        default=(DELTA_BIN))
+    parser.add_argument('--bins', type=int, \
+        default=(BINS))
+    args = parser.parse_args()
+
+    # All useful information should come from the config file
+    # This will be source of truth for starting point for each hutch for now
+    with open(args.cfg) as f:
+        yml_dict = yaml.load(f, Loader=yaml.FullLoader)
+        det_map = yml_dict['det_map']
+        ipm_name = yml_dict['ipm']['name']
+        ipm_det = yml_dict['ipm']['det']
+        hutch = yml_dict['hutch']
+
+    # Setup data source and write directories
+    exp_run = ''.join(['exp=', args.exp, ':run=', args.run])
+    base_dir = ''.join(['/cds/data/psdm/', hutch, '/', args.exp, '/'])
```

Comment:
While I get these folders likely won't change in our filesystem a long while, can they be moved to the yaml as it seems like a configuration value and just for portability's sake?